keycloak/keycloak-admin-ui
# Realm settings(localization): Add bundle functionality

## Motivation

#683 

## Brief Description


## Verification Steps



1. Go to Realm settings -> localization
2. Click "Add message bundle" button
3. Add a key-value pair
4. Verify that the pair displays in the data table below

## Checklist:


## Additional Notes
